reflectometry/output.ort

```diff
@@ -0,0 +1,245 @@
+# # ORSO reflectivity data file | 0.1 standard | YAML encoding | https://reflectometry.org
```

Comment:
As before: Why do we include a data file in the repo?

Reply:
I added it back in cause it is now linked in the notebook. This means that the instrument scientists can see the sort of file without having to run the code/have the data.

Reply:
See #34.